docs/postgres.md

```diff
@@ -13,6 +13,24 @@ Using brew was not an option:
 - `brew install postgresql postgis` would end up with Postgres 10.x
 - `brew install postgresql@9.x` would end up with Postgres 9.x without possibility to install Postgis
 
+### Using Docker
+
+If you don't want to run a local instance of Postgres in your computer, you can run one in Docker.
+Keep in mind that you still need to have the local client tools like `psql`, `dropdb`, `createuser` still locally available.
```

Comment:
When following these steps, how are you making the CLI tools available locally then?